src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1982,6 +1982,8 @@ internal enum ErrorCode
         ERR_FileScopedNamespaceNotBeforeAllMembers = 8956,
         ERR_NoImplicitConvTargetTypedConditional = 8957,
 
+        ERR_Newline_is_not_allowed_inside_a_non_verbatim_interpolated_string = 9000,
```

Comment:
>9000 [](http://example.com/codeflow?start=79&length=4)

Needs to be a real number

Reply:
9000 is an integer. Which is one of the real numbers

Reply:
Sure, but it's not the next number in sequence here. It jumps a bunch of IDs.